14.task.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+// Чи правильне це рішення задачі по умові? 
+//(тому що результат правильний, але чи відповідає він умові? ) 
+		const array = [1, 2, 3, 4, 5, 6, 7];
+		Array.prototype.remove = function removeElement(a, b) {
+				this.splice(4, 1)
+		}
+		array.remove();
+		console.log(array);
+
+// більш лаконічний варіант: 
+// (він не відповідає умові задачі, але дає правильний результат)
+		const array = [1, 2, 3, 4, 5, 6, 7];
+		function removeElement(arr, n) {
+			return arr.filter(e => e != n )
+		}
+		console.log(removeElement(array, 5))
```

Comment:
Good, your solution returns a new array, because `filter` returns a new array

This is a solution without creating a new array.

```
const items = [1, 2, 3, 4, 5, 6, 7];

function removeElement(array, value) {
  const index = array.indexOf(value);

  if (index !== -1) {
     array.splice(index, 1); 
  }
}

removeElement(items, 3);

console.log(items);
```